### ISSUE TITLE
numbing does not look correct

### ISSUE THREAD

**Issue by zhao414** [onlyafly/number-headings-obsidian#29]
The numbering result of the following note is not correct.

```md
---
number headings: first-level 1, max 6, _.1.1
---


# h1
## 1 h1-2
### 1.1 h1-2-1
## 2 h1-3
### 2.1 h1-3-1
### 2.2 h1-3-2

# h2
## 3 h2-1
### 3.1 h2-1-1
### 3.2 h2-1-2
## 4 h2-2
```
The expected result would be:

```md
---
number headings: first-level 1, max 6, _.1.1
---

# h1
## 1 h1-2
### 1.1 h1-2-1
## 2 h1-3
### 2.1 h1-3-1
### 2.2 h1-3-2

# h2
## 1 h2-1
### 1.1 h2-1-1
### 1.2 h2-1-2
## 2 h2-2
```

**Comment by zhao414** [comment]
Hi, I made a pull request which may fix this issue.

**Comment by onlyafly** [comment]
Fixed in version 1.9.0
